transport/whisper/types/whisper.go

```diff
@@ -43,12 +50,10 @@ type Whisper interface {
 	DeleteSymKey(id string) bool
 	GetSymKey(id string) ([]byte, error)
 
-	Subscribe(f Filter) (string, error)
+	Subscribe(privateKeyID, symKeyID string, pow float64, topics [][]byte) (string, error)
```

Comment:
Looks like a struct would be better. Maybe something other than `Filter` though.

Reply:
Done. Added `SubscriptionOptions`.